### ISSUE TITLE
Replying with an image does not show the reply

### ISSUE THREAD

**Issue by Polsaker** [vector-im/element-android#6570]
### Steps to reproduce

1. Reply to a message with an image on Element Web
2. Check the message on Element Android

#### What did you expect?
I should be able to see the reply and the image, just like in element web:
![image](https://user-images.githubusercontent.com/2000719/179339403-229d0ee5-1057-45b8-9bec-81afbe0aa24d.png)

#### What happened instead?
I can only see the image and not the reply:

![image](https://user-images.githubusercontent.com/2000719/179339421-c561f9d0-def1-4785-8de1-7896bc220ce3.png)

### Your phone model

Samsung S21FE

### Operating system version

Android 12

### Application version and app store

1.4.27 from google play

### Will you send logs?

Yes

### Are you willing to provide a PR?

No

**Comment by ouchadam** [comment]
did element android support this in the past? I'm tempted to switch from defect to enhancement :thinking:

**Comment by timokoesters** [comment]
For some reason it's the opposite for me, I can see the original message, but not the image reply.

**Comment by Polsaker** [comment]
> For some reason it's the opposite for me, I can see the original message, but not the image reply.

Yeah I can reproduce that now. Probably something changed and now the reply is rendered but not the image

**Comment by K4LCIFER** [comment]
I have the same issue.

**Comment by thrashwerk** [comment]
> For some reason it's the opposite for me, I can see the original message, but not the image reply.

Still same issue on Android.

If I press on the reply and hold then the image is displayed in the opened menu.
![image](https://github.com/element-hq/element-android/assets/113341896/54653fbc-e9bd-457a-b135-356adaf07203)
![image](https://github.com/element-hq/element-android/assets/113341896/5ff24f2b-e9f3-46bb-8028-1e1f9a4b2512)

**Comment by softmix** [comment]
also #8432
